### ISSUE TITLE
[Accessibility]: Text contrast of 2.81 with its background is less than the WCAG AA minimum requirements for text of size 11px and weight of 900

### ISSUE THREAD

**Issue by thefirstartist** [carbon-design-system/carbon-charts#1463]
### Contact Details

samuelting@yahoo.com

### Environment

Automated testing tool and ruleset

### What happened? What did you expect to happen instead?

When scanning with IBM Accessibility Checker, violations found in chart "Text contrast of 2.81 with its background is less than the WCAG AA minimum requirements for text of size 11px and weight of 900"

Here's the screenshot in G100 theme from storybook:
![image](https://user-images.githubusercontent.com/40870771/199551149-5fbdb677-8c43-4f60-ac10-1f080d8cb8c2.png)

Here's the screenshot in white theme:
![image](https://user-images.githubusercontent.com/40870771/199550886-e3da1deb-1d55-4c7b-9d69-a3dd01380253.png)

We received the same text contrast violation in CPD.

### What WCAG 2.1 checkpoint does the issue violate?

The contrast ratio of text with its background must meet WCAG 2.1 AA requirements

### Version

@carbon/charts@0.41.79

In storybook (11/2/2022):
https://charts.carbondesignsystem.com/?path=/story/simple-charts-area-stacked--stacked-area-time-series

**Comment by thefirstartist** [comment]
I think the problem is in checker. When viewing it in g100 theme, the background is #161616. However, it didn't pick up the correct text color from `--cds-text-secondary` variable, instead, it uses the fallback color #525252 and returns a violation.
![image](https://user-images.githubusercontent.com/40870771/199554547-3d8560a6-91bf-4f12-b996-a077ae2963fb.png)

**Comment by tombrunet** [comment]
@thefirstartist Can you update the extension and try again? The UI of the latest extension doesn't look exactly like that. When I run against Storybook using the latest, those errors do not trigger.

**Comment by thefirstartist** [comment]
@tombrunet I checked the Chrome store and see that I am using the latest version of the extension. I have removed it and installed it again and got exactly the same result. Here's the version I got from Chrome store:

![image](https://user-images.githubusercontent.com/40870771/199825058-12581389-795a-4011-8cbc-64e54fa8559c.png)

**Comment by tombrunet** [comment]
Did you scan again with that version? The screeenshot you included was not from that version.

**Comment by tombrunet** [comment]
And using the "latest" ruleset in the options?

**Comment by thefirstartist** [comment]
Yes, I have scanned. Here's the one I just did:

![image](https://user-images.githubusercontent.com/40870771/199826268-d1c07f20-cd68-419e-a8ed-181368d2bde1.png)

**Comment by thefirstartist** [comment]
How do I get the latest ruleset?

**Comment by tombrunet** [comment]
Okay - now I see why I'm seeing different UI and results. 
1) Click the "Elements" main tab, then go to the "Accessibility Checker" subtab. That's much easier to correlate the issues with the relevant DOM nodes. Those issues aren't relevant to the items that you showed.
2) When checking components in Storybook, click on the top-right icon that pops out that story into it's own page. Otherwise, you're scanning a bunch of Storybook junk that isn't relevant.

The error that you're getting isn't from the chart. It's from the headings in Storybook ("Docs", "Simple Charts", etc). Those are low contrast.

**Comment by thefirstartist** [comment]
@tombrunet Ah, that's good to know! Thanks for the clarification. I'll close this issue for now.

**Comment by theiliad** [comment]
Going to close this out, I'm not able to see this violation using the scanner anymore.
